README.md

```diff
@@ -1,21 +1,12 @@
-# OSP Director 13 Integration with Nuage
+# OSP Director Integration with Nuage
 
-## Mellanox CX-5 OVS Offload : Initial release
+#### Mellanox CX-5 OVS Offload
 
-## Note: There has been change in the documentation release format starting from OSPD13 + Nuage 5.4.1.
-
-Please see the nuage release document in release folder.
-
-For Queens + Nuage release 5.4.1 document please refer : [README.rst](Documentation/5.4.1/README.rst)
-
-This repository provides information, architecture and deployment steps for integrating OSP Director with Nuage VSP.
+This repository provides information, architecture and deployment steps for integrating Red Hat OSP Director with Nuage VSP.
 
 The repository includes:
 
-1. image-patching: overcloud-full.qcow2 image patching resources
-
-2. openstack-tripleo-heat-templates: Heat-template changes required for OSP Director and Nuage integration
-
-3. generate-cms-id: CMS ID generation resources
-
-4. documentation: The details of integration, architecture and deployment steps can be found under Documentation folder
\ No newline at end of file
+1. **image-patching**: overcloud-full.qcow2 image patching resources.
+2. **nuage-tripleo-heat-templates**: Heat-template changes required for OSP Director and Nuage integration
```

Comment:
can we reword it this way
Heat templates requried to for Nuage Integration with OSP Director?

Reply:
Sure.